examples/translation/t5/README.md

```diff
@@ -0,0 +1,51 @@
+***This script evaluates the [T5 Model](https://arxiv.org/pdf/1910.10683.pdf) ``t5-base`` on the English to German WMT dataset. Please note that the results in the paper were attained using a ``t5-base`` model fine-tuned on translation, so that results will be slightly worse here***
```

Comment:
1. Suggest changing "the [T5 Model](https://arxiv.org/pdf/1910.10683.pdf) ``t5-base``" to "the multitask pre-trained checkpoint for [``t5-base``](https://arxiv.org/pdf/1910.10683.pdf)
2. If you want to be more specific than "slightly", it looks like they are about 1.5 BLEU points lower than they would be with fine-tuning.